packages/ra-ui-materialui/src/form/SimpleForm.js

```diff
@@ -49,72 +56,89 @@ const sanitizeRestProps = ({
     untouch,
     valid,
     validate,
+    validating,
     _reduxForm,
     ...props
 }) => props;
 
-export class SimpleForm extends Component {
-    handleSubmitWithRedirect = (redirect = this.props.redirect) =>
-        this.props.handleSubmit(values => this.props.save(values, redirect));
+export const SimpleForm = ({
```

Comment:
I was confused when I saw that this SimpleForm doesn't include a Form. I think you should inverse the naming of the two components in this file, and export SimpleForm which would call a SimpleFormView. Also, please inverse the two components so that the reader goes from the top down.